doc/tutorials/sagemaker/sme_train_model.ipynb

```diff
@@ -0,0 +1,379 @@
+{
```

Comment:
<p>Probably, a feature which we could consider for the sagemaker extension </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>